ksql-streams/src/main/java/io/confluent/ksql/execution/streams/StreamMapValuesBuilder.java

```diff
@@ -40,7 +40,7 @@ private StreamMapValuesBuilder() {
         step.getProperties().getQueryContext()
     );
 
-    final LogicalSchema sourceSchema = step.getSource().getProperties().getSchema();
+    final LogicalSchema sourceSchema = stream.getSchema();
```

Comment:
nice - I missed this one :)

Reply:
Why is it ok to get the schema out of this step, but not the one above?

Reply:
are, got it.